src/Silk.Core.Data/MediatR/Tags/UpdateTagRequest.cs

```diff
@@ -14,34 +15,42 @@ namespace Silk.Core.Data.MediatR.Tags
     /// <param name="GuildId">The Id of the Guild</param>
     public record UpdateTagRequest(string Name, ulong GuildId) : IRequest<Tag>
     {
+        public ulong? Owner { get; init; }
```

Comment:
Sorry for this being picky on naming, `OwnerId` to match Id names in other Request objects 😁